jobs/build-podman-os.Jenkinsfile

```diff
@@ -0,0 +1,355 @@
+        // Get the list of requested architectures to build for
+        def arches = params.ARCHES.split() as Set
+        def archinfo = arches.collectEntries{[it, [:]]}
+        for (arch in archinfo.keySet()) {
+            // initialize some data
+            archinfo[arch]['session'] = ""
+        }
+
+        // By default we will allow re-using cache layers for one day.
+        // This is mostly so we can prevent re-downloading the RPMS
+        // and repo metadata and over again in a given day for successive
+        // builds.
+        def cacheTTL = "24h"
+        def force = ""
+        if (params.FORCE) {
+            force = '--force'
+            // Also set cacheTTL to 0.1s to allow users an escape hatch
+            // to force no cache layer usage.
+            cacheTTL = "0.1s"
+        }
+
+        withCredentials([file(credentialsId: 'podman-push-registry-secret', variable: 'REGISTRY_SECRET')]) {
+            stage('Build Layered Image') {
+                 parallel arches.collectEntries{arch -> [arch, {
+                     pipeutils.withPodmanRemoteArchBuilder(arch: arch) {
+                         shwrap("""
+                             cosa remote-build-container \
+                                 --arch $arch --cache-ttl ${cacheTTL} \
+                                 --git-ref $commit ${force} \
+                                 --git-url ${params.PODMAN_MACHINE_GIT_URL} \
+                                 --git-sub-dir podman-image-daily \
+                                 --repo ${params.CONTAINER_REGISTRY_STAGING_REPO} \
+                                 --push-to-registry --auth=\$REGISTRY_SECRET
+                         """)
+                     }
+                 }]}
+            }
+        }
+        // Initialize the sessions on the remote builders
+        stage("Initialize Remotes") {
+            parallel archinfo.keySet().collectEntries{arch -> [arch, {
+                if (arch != "x86_64") {
+                    pipeutils.withPodmanRemoteArchBuilder(arch: arch) {
+                        archinfo[arch]['session'] = pipeutils.makeCosaRemoteSession(
+                            env: ["OSBUILD_EXPORT_FORCE_NO_PRESERVE_OWNER":"true"],
+                            expiration: "60m",
+                            image: params.COREOS_ASSEMBLER_IMAGE,
+                            workdir: WORKSPACE,
+                        )
+                    }
+                }
+            }]}
+        }
+        stage("Init") {
+            parallel archinfo.keySet().collectEntries{arch -> [arch, {
+                pipeutils.withOptionalExistingCosaRemoteSession(
+                arch: arch, session: archinfo[arch]['session']) {
+                    shwrap("""cosa init --force https://github.com/coreos/fedora-coreos-config.git""")
+                }
+                shwrap("""cosa init --force https://github.com/coreos/fedora-coreos-config.git""")
```

Comment:
Wait, why are we doing another `cosa init` here? The one above should work fine across all arches.